src/Helpers.php

```diff
@@ -15,22 +15,33 @@ public static function escapeHtml(string $s): string
 
 	public static function isHtmlMode(): bool
 	{
-		return empty($_SERVER['HTTP_X_REQUESTED_WITH']) && empty($_SERVER['HTTP_X_TRACY_AJAX'])
+		return self::empty('HTTP_X_REQUESTED_WITH')
+			&& self::empty('HTTP_X_TRACY_AJAX')
```

Comment:
```suggestion
		return ($_SERVER['HTTP_X_REQUESTED_WITH'] ?? '') === '' && ($_SERVER['HTTP_X_TRACY_AJAX'] ?? '') === ''
```

Reply:
Chyba v analýze viz accesses the super-global variable $_SERVER

Reply:
V core logice balíku toto chování mít ale musíme. Pracovat s globálními stavy se nehodí v aplikaci, ale tady to bez toho nepůjde. Chybu musíme ignorovat nebo skipnout.